catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/CatalogRuntimeException.java

```diff
@@ -11,22 +11,18 @@
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.commands.util;
+package ddf.catalog.util.impl;
 
-public class CatalogCommandRuntimeException extends RuntimeException {
-    public CatalogCommandRuntimeException() {
-        super();
-    }
-
-    public CatalogCommandRuntimeException(String message) {
+public class CatalogRuntimeException extends RuntimeException {
+    public CatalogRuntimeException(String message) {
```

Comment:
Removing this constructor breaks ImportCommand.java, so I have left it in https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java#L96